qemu/tests/multi_disk_random_hotplug.py

```diff
@@ -273,6 +272,9 @@ def _unplug(new_devices, qdev, monitor, prefix=""):
         unplug_outs = []
         unplug_devs = []
         for device in new_devices[::-1]:    # unplug all devices
+            # No need to use cmd '__com.redhat_drive_del' to delete the disks
```

Comment:
@xutian That's for the parallel case, HMP has no command '__com.redhat_drive_del'; besides, currently, to upplug a drive, we don't need to use the command, only comd 'device_del' is enough.